### ISSUE TITLE
Expose the client as a node module

### ISSUE THREAD

**Issue by willdurand** [willdurand/pino-devtools#3]
The idea would be to bundle the client in an app so that it displays both the server and client logs in the page.

**Comment by willdurand** [comment]
@kumar303 I don't think this can work, server logs are sent as soon as we get them, but the app is loaded in the browser some time after.

The client can be imported already in an app:

```js
import PinoDevTools from 'pino-devtools/src/client';

PinoDevTools(console, WebSocket, window);
```

On solution would be to buffer the server logs until the client connects to the WebSocket server. But there is no need for WebSocket since we'll have all the logs at once.

**Comment by kumar303** [comment]
Buffering sounds like the right way to go. The interesting part to know is *what happened on the server before I got the first response*. And, yes, maybe we don't need the websocket. Removing the websocket would reduce complexity, too.

The main development case I see here is trying to look at how the initial data gets loaded. This is where all of our subtle SSR-related bugs happen.

**Comment by willdurand** [comment]
I'll give it a try, I think I can provide both options in this package.
